draft-ietf-quic-transport.md

```diff
@@ -1943,6 +1943,11 @@ the client during connection establishment with a Retry packet (see
 {{validate-retry}}) or in a previous connection using the NEW_TOKEN frame (see
 {{validate-future}}).
 
+Until the path is validated, a server SHOULD pad the Initial, Handshake, 1-RTT
+packets it sends so that their UDP payload size will be at least 1200 bytes.
```

Comment:
The server cannot pad Retry packets.

Reply:
```suggestion
Until the path is validated, a server SHOULD pad all packets it sends, other
than Retry, so that their UDP payload size will be at least 1200 bytes.
```